src/components/GoodsList.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class GoodsList extends React.Component {
+  state = {
+    goods: this.props.goodsList,
```

Comment:
https://stackoverflow.com/questions/40063468/react-component-initialize-state-from-props
don't use props in the state